utils/billgeneration.js

```diff
@@ -1,20 +1,68 @@
 /* eslint-disable import/imports-first */
 import moment from 'moment';
 import dotenv from 'dotenv-safe';
+import fs from 'fs';
 
+import React from 'react';
+import ReactPDF from '@react-pdf/renderer';
 import {
   getSolarProjectById,
   getOwnerById,
   getRateScheduleById,
   createSubscriberBill,
   updateSubscriberBill,
-  getSubscriberBillsByStatementNumber
+  getSubscriberBillsByStatementNumber,
+  getSubscriberBillById
 } from '../airtable/request';
 import getEnphaseData from './enphase';
 import getLatestBill from './utilityApi';
+import BillingTemplate from './pdf/BillingTemplate';
+
+// const ROOT_URL = 'https://1f7c24d2.ngrok.io';
+const ROOT_URL = 'https://peoplepower-node.herokuapp.com';
```

Comment:
I poked around the codebase a little and noticed that all in all we have quite a number of constants decalred in various js files. Given the way the codebase has grown perhaps it might be a good idea to now add all these constants into a `constants.js` file or something. I'd put this as a nit though

Reply:
yeah maybe. will look into